ui/app/src/services/content.ts

```diff
@@ -562,6 +610,74 @@ export function insertComponent(
   );
 }
 
+export function insertInstance(
```

Comment:
Should be using the performOperation function. Looks like it doesn't consider embedded.

Reply:
we can see this on call